src/ResourceManager/Network/Commands.Network/help/New-AzureRmIpsecPolicy.md

```diff
@@ -0,0 +1,172 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# New-AzureRmIpsecPolicy
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+```
+New-AzureRmIpsecPolicy -SALifeTimeSeconds <Int32> -SADataSizeKilobytes <Int32> -IpsecEncryption <String>
+ -IpsecIntegrity <String> -IkeEncryption <String> -IkeIntegrity <String> -DhGroup <String> -PfsGroup <String>
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```

Comment:
@henry416 you do not need to add those files - I am working on a change that will get rid of the XML help from the repository and refresh all of the markdown files (updating them with the latest changes in the code), so all of that should be taken care of when my PR gets merged